tests/system/marathon_pods_tests.py

```diff
@@ -519,7 +519,7 @@ def test_pod_with_persistent_volume():
 
     tasks = common.get_pod_tasks(pod_id)
 
-    host = tasks[0]['statuses'][0]['container_status']['network_infos'][0]['ip_addresses'][0]['ip_address']
+    host = common.running_status_network_info(tasks[0]['statuses'])['ip_addresses'][0]['ip_address']
```

Comment:
we need to know if this is based on recent breaking changes for networks... if it is, we need to know if it works for older marathon versions.   We need a solution that can be run on all supported cluster versions.

Reply:
this solution will work with old as well as new versions...